lm_eval/tasks/unitxt/task.py

```diff
@@ -41,17 +52,30 @@ def __init__(
         self,
         config: Optional[dict] = None,
     ) -> None:
+        if config is None:
+            config = {}
         assert "recipe" in config, "Unitxt task must have a 'recipe' string."
         super().__init__(
             config={
                 "metadata": {"version": self.VERSION},
-                "dataset_kwargs": {"trust_remote_code": True},
                 "dataset_name": config["recipe"],
-                "dataset_path": "unitxt/data",
             }
         )
+        self.image_decoder = datasets.Image()
         self.metrics = self.dataset["test"][0]["metrics"]
 
+    def download(self, dataset_kwargs: Optional[Dict[str, Any]] = None) -> None:
+        if is_unitxt_installed():
+            from unitxt import load_dataset
+
+            self.dataset = load_dataset(self.DATASET_NAME)
+        else:
+            self.dataset = datasets.load_dataset(
+                name=self.DATASET_NAME,
+                path="unitxt/data",
+                trust_remote_code=True,
```

Comment:
Is there some global configuration for `trust_remote_code`? If so, should we use that here vs hardcoding `trust_remote_code` to true?

Reply:
Note that the trust_remote_code is required because if unitxt is not locally installed via pip, we are downloading it from Huggingface and we must "trust" the unitxt code.    

We are gradually deprecating the download of unitxt code from HF (  In a new PR to main of LM-eval, we are removing it ).

Reply:
@yoavkatz This PR (https://github.com/opendatahub-io/lm-evaluation-harness/pull/12)  would add unitxt explicitly as a dependency, so unitxt would be available locally.
Would it be possible to change this to `False` if #12 merged before?

Reply:
cc @yhwang

Reply:
Yes. If #12 is merged, we can actually remove this HF code:

          if is_unitxt_installed():
             from unitxt import load_dataset

             self.dataset = load_dataset(self.DATASET_NAME)
         else:
             raise Exception("Please install unitxt via 'pip install unitxt'. For more information see: https://www.unitxt.ai/")

Reply:
done for both `unitxt/data` and `unitxt/metric`.